### PR TITLE
Add new changes to the press page

### PR DESCRIPTION
**Pull request by alicetragedy**
This PR adds recent changes made mostly by @anasofiapinho to the press page. 
In the future, we will add links to media and press releases, as well as updated quotes from past participants (and maybe sponsors).

### CODE REVIEW

**Comment by anasofiapinho** [comment]
I think it looks good @alicetragedy ! :)

**Comment by alicetragedy** [comment]
@anasofiapinho would you like to have the honour of merging this PR? ;)

File: about/press.html [hunk 1]
```diff
@@ -6,79 +6,66 @@
...
+      </p>
 
-        <a target="_blank" href="https://f.cloud.github.com/assets/1711357/653102/70f37298-d4cd-11e2-9a44-ab9683bf9521.jpg"><img src="https://f.cloud.github.com/assets/1711357/653102/70f37298-d4cd-11e2-9a44-ab9683bf9521.jpg" title="Linda Liukas, Co-Founder of Rails Girls, at Rails Girls Helsinki" height="150px"></a>
+      <p>
+      This year, over 90 teams applied for the program, almost doubling RGSoC 2015’s record! So far, 44 Open Source projects were accepted for this edition and 12 teams will be funded. All selected teams, including volunteer teams, will be announced at the beginning of June. Meanwhile, RGSoC has launched the <a href="http://railsgirlssummerofcode.org/blog/2016-04-22-thank-you-lets-diversify-tech">#diversifytech</a> campaign to thank all of the people who have contributed to the program.
```
Comment (carpodaster) [inline about/press.html]: Shall we change »This year« to »In 2016«?  That way the page will still be up to date for a  little longer

Comment (alicetragedy) [inline about/press.html]: sounds great @carpodaster! Good point. :) I'll do this right now if you're cool with that

**Comment by anasofiapinho** [comment]
My first PR merge! Thank you @alicetragedy! <3

**Comment by alicetragedy** [comment]
Congratulations @anasofiapinho! 🎉

**Pull request merged**